filebeat/module/nats/log/_meta/fields.yml

```diff
@@ -4,3 +4,53 @@
     Nats log files
   release: beta
   fields:
+    - name: client
+      type: group
+      description: >
+        Fields from NATS logs client.
+      fields:
+        - name: id
+          type: integer
+          description: >
+            The id of the client
+    - name: msg
+      type: group
+      description: >
+        Fields from NATS logs message.
+      fields:
+        - name: bytes
+          type: long
+          description: >
+            Size of the payload in bytes
+        - name: direction
+          type: text
+          description: >
+            The direction of the message
+        - name: type
+          type: keyword
+          description: >
+            The protocol message type
+        - name: subject
+          type: keyword
+          description: >
+            Subject name this message was received on
+        - name: sid
+          type: integer
+          description: >
+            The unique alphanumeric subscription ID of the subject
+        - name: reply_to
+          type: keyword
+          description: >
+            The inbox subject on which the publisher is listening for responses
+        - name: max_messages
+          type: integer
+          description: >
+            An optional number of messages to wait for before automatically unsubscribing
+        - name: error
```

Comment:
I wonder if we should map this to `error.message`. If we don't I suggest to at least use here `error.message` to be consistent with a mapped ECS.

@webmat Thoughts?

Reply:
I would agree in principle, but unfortunately it would be overwritten by the ingest pipeline, when there's a pipeline error.

I think we'll need to better define what goes where in the future, but for now I think a custom field for an error message from the source is fine.